packages/runtime/client/rest/src/contract/endpoints/collection.ts

```diff
@@ -41,7 +42,14 @@ import {
 } from "../rolesConfigurations.js";
 
 import { ContractId, ContractIdGuard } from "@marlowe.io/runtime-core";
-import { ItemRange, Page, PageGuard } from "../../pagination.js";
+import {
+  ItemRange,
+  ItemRangeGuard,
+  Page,
+  PageGuard,
+} from "../../pagination.js";
+import { ContractGuard } from "@marlowe.io/language-core-v1/contract";
+import { RoleNameGuard } from "@marlowe.io/language-core-v1/participants";
```

Comment:
these should be 
`import * as G from "@marlowe.io/language-core-v1/guards`

G.Contract
G.RoleName

Reply:
This may remove the need for some of the exported modules.